solvers/lightning.py

```diff
@@ -9,6 +9,8 @@
 class Solver(BaseSolver):
     name = 'Lightning'
 
+    # conda install -c conda-forge sklearn-contrib-lightning
+    # did not install lightning with CDClassifier ...
```

Comment:
Yes because lightning is not shipped on conda.
If you look at the requirement, it is a `pip` requirement so it is installed as `pip install git+https://github.com/scikit-learn-contrib/lightning.git`.